pkg/domain/podman/router.go

```diff
@@ -0,0 +1,234 @@
+package podman
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/client/podman"
+	"github.com/skupperproject/skupper/pkg/qdr"
+)
+
+type RouterEntityManagerPodman struct {
+	cli       *podman.PodmanRestClient
+	container string
+}
+
+func NewRouterEntityManagerPodman(cli *podman.PodmanRestClient) *RouterEntityManagerPodman {
+	return NewRouterEntityManagerPodmanFor(cli, types.TransportDeploymentName)
+}
+
+func NewRouterEntityManagerPodmanFor(cli *podman.PodmanRestClient, container string) *RouterEntityManagerPodman {
+	return &RouterEntityManagerPodman{
+		cli:       cli,
+		container: container,
+	}
+}
+
+func (r *RouterEntityManagerPodman) exec(cmd []string) (string, error) {
+	return r.cli.ContainerExec(r.container, cmd)
+}
+
+func (r *RouterEntityManagerPodman) CreateSslProfile(sslProfile qdr.SslProfile) error {
+	cmd := qdr.SkmanageCreateCommand("sslProfile", sslProfile.Name, sslProfile)
+	if _, err := r.exec(cmd); err != nil {
+		return fmt.Errorf("error creating sslProfile %s - %w", sslProfile.Name, err)
+	}
+	return nil
+}
+
+func (r *RouterEntityManagerPodman) DeleteSslProfile(name string) error {
+	cmd := qdr.SkmanageDeleteCommand("sslProfile", name)
+	if _, err := r.exec(cmd); err != nil {
+		return fmt.Errorf("error deleting sslProfile %s - %w", name, err)
+	}
+	return nil
+}
+
+func (r *RouterEntityManagerPodman) CreateConnector(connector qdr.Connector) error {
+	cmd := qdr.SkmanageCreateCommand("connector", connector.Name, connector)
+	if _, err := r.exec(cmd); err != nil {
+		return fmt.Errorf("error creating connector %s - %w", connector.Name, err)
+	}
+	return nil
+}
+
+func (r *RouterEntityManagerPodman) DeleteConnector(name string) error {
+	cmd := qdr.SkmanageDeleteCommand("connector", name)
+	if _, err := r.exec(cmd); err != nil {
+		return fmt.Errorf("error deleting sslProfile %s - %w", name, err)
+	}
+	return nil
+}
+
+func (r *RouterEntityManagerPodman) QueryConnections(routerId string, edge bool) ([]qdr.Connection, error) {
+	cmd := qdr.SkmanageQueryCommand("connection", routerId, edge, "")
+	var data string
+	var err error
+	if data, err = r.exec(cmd); err != nil {
+		return nil, fmt.Errorf("error querying connections - %w", err)
+	}
+	var connections []qdr.Connection
+	ioutil.WriteFile("/tmp/baddata", []byte(data), 0644)
```

Comment:
is this write necessary? where does it get referenced?

Reply:
that is a debug item left in the code, removing...